packages/function-demethodize/package.json

```diff
@@ -10,6 +10,7 @@
       "default": "./index.mjs"
     }
   },
+  "types": "index.d.ts",
```

Comment:
This package does not yet have a type definition file

Reply:
@angus-c 
Thanks, fixed.